docs/insights-archive-sample/config/workload_info.json

```diff
@@ -0,0 +1,7166 @@
+{
+  "pods": 5156,
+  "imageCount": 171,
+  "images": {
```

Comment:
I am failing to see why the number of images is so small (in this struct) in my testing clusters (not many external stuff deployments and usually only 3 items in the struct). 

Well the client can't find them in https://github.com/openshift/insights-operator/pull/315/files#diff-7f69d2c3dea3690849da339327364050f1b27a8de85fafc8ee832c0faad051c5R336. 
I am wondering if it's an intention, because it can't find for example the image of apiserver deployment/pods.

Reply:
These would only be images imported on the cluster, vs images directly referenced.  So yes, not all images would be included.  The goal is to use the image api to selectively get minimal info about the image - if we don't have the image metadata, that's ok.  Down the road, components like the image scanning operator might also report a subset of their info, enriching info about the images.  But for now, the simplest possible structure gathers data that can be correlated.

Reply:
Thanks